src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothHid.cs

```diff
@@ -78,6 +78,199 @@ public void Disconnect()
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.</exception>
+        /// or when the connection attempt to the remote device fails.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void DeactivateDevice()
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                int ret = BluetoothHidImpl.Instance.DeactivateHidDevice();
+                if (ret != (int)BluetoothError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to DeactivateHidDevice - " + (BluetoothError)ret);
+                    BluetoothErrorFactory.ThrowBluetoothException(ret);
+                }
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+        }
+
+        /// <summary>
+        /// Connects to the remote device.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.</exception>
+        /// or when the connection attempt to the remote device fails.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public void ConnectDevice()
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                int ret = BluetoothHidImpl.Instance.ConnectHidDevice(RemoteAddress);
+                if (ret != (int)BluetoothError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to ConnectHidDevice - " + (BluetoothError)ret);
+                    BluetoothErrorFactory.ThrowBluetoothException(ret);
+                }
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+        }
+
+        /// <summary>
+        /// Disconnects to the remote device.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.</exception>
+        /// or when the connection attempt to the remote device fails.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public void DisconnectDevice()
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                int ret = BluetoothHidImpl.Instance.DisconnectHidDevice(RemoteAddress);
+                if (ret != (int)BluetoothError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to DisconnectHidDevice - " + (BluetoothError)ret);
+                    BluetoothErrorFactory.ThrowBluetoothException(ret);
+                }
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+        }
+
+        /// <summary>
+        /// Sends the mouse event data to the remote device.
```

Comment:
I'm hard to understand this API behavior,

It is a object to handle HID device and usually Mouse event was sent from HID Device
What is remote device?
Who is HID Device?
Is it possible to send a mouse event to HID Device?